app/views/professional_experiences/index.html.erb

```diff
@@ -0,0 +1,48 @@
+<div id="page-top" class="row">
+  <div class="container-fluid home">
+    <div class="card mb-3">
+      <div class="card-header">
+        <i class="fa fa-table"></i>
+        <%= t('activerecord.models.professinal_experience.other') %>
+      </div>
+      <div class="card-body">
+        <div class="table-responsive">
+          <table class="table table-hover table-bordered users-evaluation" id="dataTable">
+            <thead>
+              <tr>
+                <th><%= t('activerecord.attributes.professional_experience.company') %></th>
+                <th><%= t('activerecord.attributes.professional_experience.position') %></th>
+                <th><%= t('activerecord.attributes.professional_experience.start_date') %></th>
+                <th><%= t('activerecord.attributes.professional_experience.end_date') %></th>
+                <th></th>
+              </tr>
+            </thead>
+            <tbody>
+              <% if @professional_experiences.empty?  %>
+                <tr>
+                  <td colspan="4"><%= t('professional_experience.not_found') %></td>
+                </tr>
+              <% end %>
+              <% @professional_experiences.each do |experience| %>
+                <tr>
+                  <td><%= experience.company %></td>
+                  <td><%= experience.position %></td>
+                  <td><%= l(Date.parse(experience.start_date), format: :short) %></td>
```

Comment:
why the date parse, aren't start_date and end_date already a date?

Reply:
It's saved as a string. Initially, I tried to parse and it threw and error saying that in can't format strings.